apps/Standalone/src/designer/app/AzureLogicAppsDesigner/laDesignerConsumption.tsx

```diff
@@ -280,23 +298,12 @@ const DesignerEditorConsumption = () => {
             workflowId={workflowDefinitionId}
             runInstance={runInstanceData}
           >
-            <div style={{ display: 'flex', flexDirection: 'column', height: 'inherit', width: 'inherit' }}>
-              <DesignerCommandBar
-                id={workflowId}
-                saveWorkflow={saveWorkflowFromDesigner}
-                discard={discardAllChanges}
-                location={canonicalLocation}
-                isReadOnly={readOnly}
-                isDarkMode={isDarkMode}
-                isDesignerView={designerView}
-                showConnectionsPanel={showConnectionsPanel}
-                enableCopilot={() => {
-                  dispatch(setIsChatBotEnabled(!showChatBot));
-                }}
-                switchViews={handleSwitchView}
-                saveWorkflowFromCode={saveWorkflowFromCode}
+            <div style={{ display: 'flex', flexDirection: 'row', height: 'inherit' }}>
```

Comment:
NIT: no need to set flexDirection as row, I think the default for display flex is row